package.json

```diff
@@ -31,5 +31,8 @@
     },
     "workspaces": [
         "packages/*"
-    ]
+    ],
+    "dependencies": {
+        "ui": "github:dhis2/ui-builds"
```

Comment:
Shouldn't this be in the app's package.json?

Reply:
Yes.